x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/StartDatafeedAction.java

```diff
@@ -194,6 +195,10 @@ public DatafeedParams(StreamInput in) throws IOException {
             startTime = in.readVLong();
             endTime = in.readOptionalLong();
             timeout = TimeValue.timeValueMillis(in.readVLong());
+            if (in.getVersion().onOrAfter(Version.CURRENT)) {
```

Comment:
It seems like this should be a specific version, not `Version.CURRENT`.  Once we get to a mixed version 7.2/7.3 cluster surely it would be OK for the master node to be on 7.2?

Reply:
CURRENT is a placeholder. This is to be merged into a feature branch that will merge into 6.x at some point in the future into in an unknown version. There is no version constant > 6.5.0 so it's CURRENT for now. OpenJobAction is the same, I'll add a checkbox to the meta issue so this isn't forgotten